tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -2789,3 +2789,229 @@ def _(
         assert output.dtype == torch.bfloat16, "CuTe DSL fp8 bmm output dtype must be bf16"
         assert output.shape == (batch_size, m,
                                 n), "CuTe DSL fp8 bmm output shape is incorrect"
+
+    # ========================================================================
+    # CuTE DSL Top-K Operations
+    # ========================================================================
+
+    import cutlass
```

Ruff (0.15.2)

[error] Redefinition of unused `cutlass` from line 306: `cutlass` redefined here

Remove definition: `cutlass`

(F811)